extensions/shared/src/main/java/app/revanced/extension/shared/patches/client/AppClient.kt

```diff
@@ -281,6 +322,7 @@ object AppClient {
             val CLIENT_ORDER_TO_USE_YOUTUBE: Array<ClientType> = arrayOf(
                 ANDROID_VR,
                 ANDROID_UNPLUGGED,
+                ANDROID_MUSIC,
```

Comment:
It doesn't look good

To get valid streaming data from `ANDROID_MUSIC` clients, two conditions must be met:
1. The user must be logged in
2. YouTube Music must be available in the user's country

The client array for YouTube Music, `CLIENT_ORDER_TO_USE_YOUTUBE_MUSIC`, may contain the `ANDROID_MUSIC` client for the following reasons:
1. If the user is using a patched YouTube Music, it means the user is logged in (and YouTube Music does not have an Incognito mode).
2. If the user is using a patched YouTube Music, it means the user is located in a country that supports YouTube Music